zilliqa/src/consensus.rs

```diff
@@ -1754,6 +1730,12 @@ impl Consensus {
             return Ok(None);
         }
 
+        if let Ok(None) = self.get_block(&new_view.qc.block_hash) {
```

Comment:
```suggestion
        if self.get_block(&new_view.qc.block_hash)?.is_none() {
```
Maybe I'm nitpicking, but the current version will silently ignore an `Err` from `get_block` which I don't think makes sense.

Reply:
No that's fair. Updated